authors/apps/authentication/serializers.py

```diff
@@ -15,25 +15,26 @@ class RegistrationSerializer(serializers.ModelSerializer):
         min_length=8,
         write_only=True
     )
-
     # The client should not be able to send a token along with a registration
     # request. Making `token` read-only handles that for us.
 
     class Meta:
         model = User
         # List all of the fields that could possibly be included in a request
         # or response, including fields specified explicitly above.
-        fields = ['email', 'username', 'password']
+        fields = ['email', 'username', 'password', 'message']
 
     def create(self, validated_data):
-        # Use the `create_user` method we wrote earlier to create a new user.
+        # Use (the `create_user` method we wrote earlier to create a new user.
```

Comment:
Kindly remove the bracket you put in this comment by mistake.

Reply:
thanks